unwrap.go

```diff
@@ -20,14 +20,14 @@ func UnwrapFrame(err error) (frame runtime.Frame, inner error, ok bool) { //noli
 		return runtime.Frame{}, err, false
 	}
 
-	wrapErr := errors.Unwrap(err)
+	inner = errors.Unwrap(err)
```

Comment:
I agree that inner is a better name. `wrapErr` was a silly play on `wrapper`, but I still get to giggle when any error name's last syllable can be pronounced `err`. 😄